code/lib/core-common/package.json

```diff
@@ -52,7 +52,7 @@
     "@yarnpkg/libzip": "2.3.0",
     "chalk": "^4.1.0",
     "cross-spawn": "^7.0.3",
-    "esbuild": "^0.18.0",
+    "esbuild": "^18.0.0 || ^19.0.0 || ^0.20.0",
```

Comment:
Typo?

`18.0.0` -> `0.18.0`
`19.0.0` -> `0.19.0`

Reply:
it is a typo! 👀  this was added by @ndelangen i think, probably trying to account for mixed versions in dependency trees. should be easy enough to fix at least